### DISCUSSION TITLE
Enforce execution of two jobs on Same Background Server

### DISCUSSION THREAD

**Discussion by mrinalsharma** [jobrunr/jobrunr#195] [Q&A] [answered]
I have a usecase where a long-running Job needs to start and later on stopped.

- Job 1(Create)
-  Job is Created on ServerA and resources are allocated on that server.
 

- Job 2(Stop)
- Stop the job1 created earlier on Server A and clean resources on the background server

Both these Jobs need to be picked by the same server A. How can we enforce it?

**Comment by rdehuyss** [comment, marked as answer]
This is possible in the pro version using server tags.

See https://www.jobrunr.io/en/documentation/pro/server-tags/.
